web/src/components/filter/ReviewFilterGroup.tsx

```diff
@@ -682,12 +685,11 @@ function ShowMotionOnlyButton({
       <div className="block md:hidden">
         <Button
           size="sm"
-          variant="secondary"
           className={`duration-0 ${motionOnlyButton ? "bg-selected hover:bg-selected" : "bg-secondary hover:bg-secondary/80"}`}
           onClick={() => setMotionOnlyButton(!motionOnlyButton)}
         >
           <FaRunning
-            className={`${motionOnlyButton ? "fill-primary dark:fill-primary-foreground" : "text-muted-foreground"}`}
+            className={`${motionOnlyButton ? "text-background dark:text-primary" : "text-secondary-foreground"}`}
```

Comment:
```suggestion
            className={`${motionOnlyButton ? "text-white" : "text-secondary-foreground"}`}
```

Reply:
Didn't want to use `text-white` so that any themes can have unique colors later.

Reply:
but right now we are putting this on top of selected background. IMO any theme that changes the selected color white will probably look good on it. If not then we should create selected-foreground and make that white

Reply:
I added a PR onto this branch for that change https://github.com/blakeblackshear/frigate/pull/10914